pkg/web/client_test.go

```diff
@@ -74,3 +74,8 @@ func TestVersionFromManifest(t *testing.T) {
 	require.Equal(t, openAPISchema, version.OpenAPISchema)
 	require.Equal(t, cogConfig, version.CogConfig)
 }
+
+func TestVersionURLErrorWithoutR8IMPrefix(t *testing.T) {
+	_, err := newVersionURL("docker.com/thing/thing")
+	require.Error(t, err)
+}
```

Comment:
We should have a test for `thing/thing`, too, shouldn't we?

Reply:
This is covered by the test for component lengths: [d74b018](https://github.com/replicate/cog/pull/2146/commits/d74b018d5783ccaafcfd3a38acd247b944720aca)